spec/factories/user_admin.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :admin, class: 'User' do
```

Comment:
admin is still `User`, so you could have only `spec/factories/user.rb` file with all factories related to User model

Reply:
solved - removed junk file